aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -177,7 +177,7 @@ pub(crate) struct ValueType {
     /// Wrapping the types used for testing to add TransactionWrite trait implementation (below).
     bytes: Option<Bytes>,
```

Comment:
I tried and it's annoying to change, because sometimes we access these fields directly. The whole testing infra here needs some love, for now added a TODO so we remember this improvement.